nio/api.py

```diff
@@ -2033,3 +2033,32 @@ def put_room_alias(
         return ("PUT",
                 Api._build_path(path, query_parameters),
                 Api.to_json(content),)
+
+    @staticmethod
+    def update_room_topic(
```

Comment:
Setting the topic doesn't have a separate API endpoint, it uses the [`/room/$roomId/state`](https://spec.matrix.org/v1.2/client-server-api/#put_matrixclientv3roomsroomidstateeventtypestatekey) endpoint which we already support.

Reply:
Fixed in e0fdf67.